<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectCodecs</class>
 <widget class="QDialog" name="SelectCodecs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="MPEG1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>175</width>
     <height>43</height>
    </rect>
   </property>
   <property name="text">
    <string>MPEG1</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="MPEG2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>175</width>
     <height>43</height>
    </rect>
   </property>
   <property name="text">
    <string>MPEG2</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="H261">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>H261</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="MJPEG">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>175</width>
     <height>43</height>
    </rect>
   </property>
   <property name="text">
    <string>MJPEG</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="H265">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>0</y>
     <width>175</width>
     <height>43</height>
    </rect>
   </property>
   <property name="text">
    <string>H265</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="H264">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>0</y>
     <width>175</width>
     <height>43</height>
    </rect>
   </property>
   <property name="text">
    <string>H264</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SelectCodecs</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SelectCodecs</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
